simulation/test/realtime_loop_test.cpp

```
@@ -0,0 +1,333 @@
+    t.predicate("pressured run completed", pressured.completed,
+                "abortReason=\"" + pressured.abortReason + "\"");
+    t.equalU64("pressured run emitted every declared frame",
+               pressured.framesAccepted, kFrames);
+    // Frame 0 is the initial state: it consumes ONE integrator step, not 200,
+    // so its cost is not comparable and on a fast machine it can land inside
+    // even a 1 us budget. The deterministic claim is about the STEPPED frames,
+    // each of which is 200 implicit DAE steps with a KKT solve apiece --
+    // measured at ~9 ms, four orders of magnitude over the budget.
+    {
+        std::size_t steppedDegraded = 0, steppedTotal = 0;
+        for (std::size_t i = 1; i < pressuredSink.frames().size(); ++i) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)